files/jquery.query-builder/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "bower",
+  "name": "jQuery-QueryBuilder",
+  "repo": "mistic100/jQuery-QueryBuilder",
+  "files": {
+    "basePath": "dist/",
+    "include": ["**/*"]
```

Comment:
What does "I think" means ? is this a requirement for acceptance or a technical requirement ?